packages/core/src/focus-manager/internal/findAllTabbableElements.ts

```diff
@@ -1,4 +1,4 @@
-import { ownerWindow } from "../../utils";
+import { ownerWindow } from "../../utils/ownerWindow";
```

Comment:
?

Reply:
Ah yes, I auto-imported this before adding the export